### PR TITLE
Bug 1199872 - Does not play sound until volumn change occured r=gasolin

### PR DESCRIPTION
**Pull request by scottwu**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=36de6ede1f75e345f5dbab4d684429e039861311

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=9e96dda4caf577d35c3df8bebc238977e51c952c

File: apps/settings/test/unit/panels/sound/slider_handler_test.js [hunk 1]
```diff
@@ -183,6 +182,28 @@ suite('Sound > SliderHandler', function() {
     });
   });
 
+  suite('_setVolumn', function() {
```
Comment (gasolin) [inline apps/settings/test/unit/panels/sound/slider_handler_test.js]: _setVolume

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=3cf651618bdbc391c096b065a7c746ed1ee4b3e0

**Pull request merged**